docs/content/_index.md

```diff
@@ -0,0 +1,48 @@
+---
+title: "Docs Home"
+description: "OSM Docs Home"
+type: docs
+---
+
+## Overview
+
+OSM runs an Envoy based control plane on Kubernetes, can be configured with SMI APIs, and works by injecting an Envoy proxy as a sidecar container next to each instance of your application. The proxy contains and executes rules around access control policies, implements routing configuration, and captures metrics. The control plane continually configures proxies to ensure policies and routing rules are up to date and ensures proxies are healthy.
+
+## Core Principles
+* Simple to understand and contribute to
+* Effortless to install, maintain, and operate
+* Painless to troubleshoot
+* Easy to configure via Service Mesh Interface (SMI)
+
+## Features
+* Easily and transparently configure traffic shifting for deployments
+* Secure service to service communication by enabling mTLS
+* Define and execute fine grained access control policies for services
+* Observability and insights into application metrics for debugging and monitoring services
+* Integrate with external certificate management services/solutions with a pluggable interface
+* Onboard applications onto the mesh by enabling automatic sidecar injection of Envoy proxy
+
+## Install
+
+### Prerequisites
+- Kubernetes cluster running Kubernetes v1.15.0 or greater
+- kubectl current context is configured for the target cluster install
+  - ```kubectl config current-context```
+
+### Get the OSM CLI
+
+The simplest way of installing Open Service Mesh on a Kubernetes cluster is by using the `osm` CLI.
+
+Download the `osm` binary from the [Releases page](https://github.com/openservicemesh/osm/releases). Unpack the `osm` binary and add it to `$PATH` to get started.
+```shell
+sudo mv ./osm /usr/local/bin/osm
+```
+
+### Install OSM
+```shell
+$ osm install
+```
+
+![OSM Install Demo](https://github.com/openservicemesh/osm/raw/main/img/osm-install-demo-v0.2.0.gif "OSM Install Demo")
+
+See the [installation guide](docs/installation_guide.md) for more detailed options.
```

Comment:
This link doesn't work in the preview and it seems like it wouldn't work in general - assuming this is being generated from the code repo, do we need to change it to a fully qualified URL so it can work from here?

Reply:
Removed this link for now so that a broken link is not introduced.

---

I'd like to keep this PR focused on generating the website (config and theme) and have content edits to the docs be added separately via follow up PRs & Issues. Discussed with @phillipgibson, there are edits that should happen and I think those are best handled by the OSM team, before this website be made accessible to the public.

### Recommended follow-up edits: 

- Create a proper landing page - the current `index.md` page referenced here is temporary (copied from the OSM readme), a tailored Docs welcome / intro is needed to best guide project users 
- Content needs to be reorganized - right now only 1 file exists in the `docs/content` site dir. The rest of the content should be reviewed and moved into the required structure, and grouped into subsections as needed `*`
- Review the metadata - the existing docs need to have Hugo frontmatter added to render properly for the web `*`

I had carried out some of this work via this PR, but it was problematic as this PR is already large without any docs files edits - there were a lot of rebase conflicts from other changes in flight. This PR would in turn then cause a lot of other rebase headaches for other open PRs. Hence, I removed those edits and reduced the size of this PR.

`*` The website README that is introduced in this PR provides further details on the ideal structure and the necessary front matter metadata that should be used.